packages/thirdweb/src/react/web/ui/ConnectWallet/screens/Buy/swap/FiatValue.tsx

```diff
@@ -42,6 +42,8 @@ export function FiatValue(
   }
 
   return cryptoToFiatQuery.data?.result ? (
-    <Text {...props}>${formatNumber(cryptoToFiatQuery.data.result, 2)}</Text>
+    <Text {...props}>
+      ${formatNumber(cryptoToFiatQuery.data.result, 2).toFixed(2)}
```

Comment:
Doesn't the 2 param inside formatNumber already does it?

Reply:
It doesn't, format number just rounds it but leaves off trailing zeros (it returns a number)

Reply:
Ah I see